ert_gui/gert_main.py

```diff
@@ -33,22 +37,27 @@
 from ert_gui.tools.plugins import PluginHandler, PluginsTool
 from ert_gui.tools.run_analysis import RunAnalysisTool
 from ert_gui.tools.workflows import WorkflowsTool
-from ert_shared.feature_toggling import FeatureToggling
 
-import os
 from res.enkf import EnKFMain, ResConfig
 from res.util import ResLog
 
-import res
 import ecl
-import sys
-import time
 
 
 def run_gui(args):
     app = QApplication([])  # Early so that QT is initialized before other imports
     app.setWindowIcon(resourceIcon("application/window_icon_cutout"))
     res_config = ResConfig(args.config)
+
+    # Create logger inside function to make sure all handlers have been added to the root-logger.
+    logger = logging.getLogger(__name__)
+    logger.info(
+        "Running ERT GUI...",
```

Comment:
We are already logging this information else where, should we change the wording here?

Reply:
"Logging forward model jobs when starting GUI"?

Reply:
Not very important, but was thinking we did not need to GUI information, but will leave it up to you.